### PR TITLE
My mobile version of portifolio 

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
> First update
 - Add toolbars
 - headline section
 - social icons and more-button

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 50px 0;
+
+  /* background: url(../images/svgs/Frame1.png);
+       background-repeat: no-repeat;
+       background-position: right top;
+       opacity: 0.7;
+*/
```
Comment (Whoistolu) [inline assets/css/style.css:100]: - [ ] [OPTIONAL] Kindly remove all unused lines of code in order to have a clean code-base.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  --neutral-icons: #36b37f;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  width: 100%;
+  background-color: var(--neutral);
+  padding: 10px;
+
+  /* background: url(Overlay.png) 50% 0% no-repeat; */
```
Comment (Whoistolu) [inline assets/css/style.css:23]: - [ ] [OPTIONAL] Kindly remove all unused lines of code in order to have a clean code-base.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez ,

Good job so far!
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

### To Highlight clap
✔️ README file is professional
✔️ Pull Request is descriptive
✔️ Proper Gitflow

- [x] [REQUIRED] I think it will be nice to add the background image in the section to your design as provided in the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). This will make your design look like the one provided in the template.
- [ ] [REQUIRED] Just as I have stated above, kindly adhere to the (margins, fonts, and font sizes) You have not applied the appropriate margins as provided in the figma template, you need to adhere to the fonts and their sizes as well.
### Your design
your-design

### Figma design
figma

- [x] [REQUIRED] Please ensure that the first letter of your commit messages starts with a capital letter, this is a practice that will groom your professionalism.
### Your commit messages
your-commits

### Expected Commit message format
proper-commit-messages




_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: INVALID  ❌ 

### Hello @ ✋🏼 
#### Pull requests include many files. You should  delete unused files , images in the repo. Right now ı cant review it.
image


#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏
coding


___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by kinginthenorthcodez** [comment]
> # STATUS: INVALID ❌
> ### Hello @kinginthenorthcodez ✋🏼
> #### Pull requests include many files. You should delete unused files, images in the repo. Right now ı cant review it.
> image
> 
> #### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> ### Cheers and Happy coding! 💻 👏
> coding
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
> 
> _If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
> 
> [GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)
> 
> _Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_
Hi @ahmetbozaci ok sure thing, and how do I do just that?

**Comment by kinginthenorthcodez** [comment]
@ahmetbozaci  how should i go about deleting the files for a new pull request?

**Comment by ahmetbozaci** [comment]
@kinginthenorthcodez I see you have a node modules folder in the repository. You can add them in the gitignore file. Right now they are already pushed. 

- [ ] So please check this [How to Gitignore (untrack) files already pushed to GitHub repository.](https://dev.to/ademola_isr/how-to-gitignore-untrack-files-already-pushed-to-github-repository-1h96)

**Comment by kinginthenorthcodez** [comment]
@ahmetbozaci  ok let do just, had already added the node_modules file .gitignore when i pushed a second time after my first reviewer noticed it

**Comment by kinginthenorthcodez** [comment]
Right i will just delete it for now

**Pull request merged**